platform-sdk/platform-apps/demos/CryptocurrencyDemo/src/main/java/com/swirlds/demo/crypto/CryptocurrencyDemoState.java

```diff
@@ -380,76 +385,14 @@ public void init(
             @NonNull final Platform platform,
             @NonNull final InitTrigger trigger,
             @Nullable final SoftwareVersion previousSoftwareVersion) {
-        this.platform = (SwirldsPlatform) platform;
+        super.init(platform, trigger, previousSoftwareVersion);
 
+        this.platform = (SwirldsPlatform) platform;
         if (trigger == InitTrigger.GENESIS) {
             genesisInit();
         }
     }
 
-    @Override
-    public void serialize(final SerializableDataOutputStream out) throws IOException {
```

Comment:
Not sure how important this particular state implementation is, but this serialization/deserialization logic cannot be just removed. However, this class may be no longer relevant, and it doesn't make sense spend too much time fixing it